docs/guides/main.rst

```diff
@@ -139,6 +146,11 @@ place to start::
     directory: ~/music
     library: ~/data/musiclibrary.db
 
+For Windows user, this would look like::
+
+    directory: "D:\\My Music Files"
+    library: "D:\\My Music Files\\Musiclibrary.db"
```

Comment:
The quotes and double-backslashes aren't actually necessary—they are harmless, but I think we can recommend that people don't need to do that, thanks to YAML's elegant syntax.